source/apiVolontaria/volunteer/tests/tests_view_Participations.py

```diff
@@ -178,6 +177,99 @@ def test_create_new_participation(self):
             'attributes : {0}'.format(attributes),
         )
```

Comment:
It could be a good thing to check if it's actually the good used id since you changed this logic.